js/markupBlog.js

```diff
@@ -0,0 +1,328 @@
+
+const createButton = (classBtn) => {
+  const btn = document.createElement("button");
+  btn.className = classBtn;
+  btn.setAttribute("type", "button");
+
+  return btn;
+};
+
+const markupDescription = (value) => {
+  const div = document.createElement("div");
+  const icon = document.createElement("div");
+  const date = document.createElement("p");
+  const img = document.createElement("img");
+  const oval = document.createElement("p");
+
+  div.className = "comment";
+  icon.className = "comment_icon";
+  date.className = "latest-posts__comment";
+  oval.className = "oval";
+
+  img.setAttribute("src", "./images/atoms/a-icon-comment.svg");
+  img.setAttribute("alt", "icon");
+
+  const time = date.cloneNode(false);
+  const text = date.cloneNode(false);
+
+  date.textContent = value.date;
+  time.textContent = value.time;
+  text.textContent = value.commetn;
+
+  icon.append(img);
+  div.append(date);
+  div.append(oval);
+  div.append(time);
+  div.append(oval.cloneNode(false));
+  div.append(icon);
+  div.append(text);
+
+  value.stars
+    ? value.stars.forEach((elem) => {
+        const divIconStar = document.createElement("div");
+        const imgIconStar = document.createElement("img");
+
+        divIconStar.className = "star_icon";
+        imgIconStar.setAttribute("src", elem);
+        imgIconStar.setAttribute("alt", "icon");
+
+        divIconStar.append(imgIconStar);
+        div.append(divIconStar);
+      })
+    : null;
+
+  return div;
+};
+
+const markupSocialLink = () => {
+  const ul = document.createElement("ul");
+  ul.className = "social_list";
+
+  data.social.forEach((elem) => {
+    const li = document.createElement("li");
+    const a = document.createElement("a");
+    const div = document.createElement("div");
+    const img = document.createElement("img");
+
+    li.className = "social_item";
+    a.className = "social_link";
+    div.className = "social_icon";
+
+    a.setAttribute("href", elem.href);
+    a.setAttribute("target", "_blank");
+    img.setAttribute("src", elem.icon);
+    img.setAttribute("alt", "icon");
+
+    div.append(img);
+    a.append(div);
+    li.append(a);
+    ul.append(li);
+  });
+  return ul;
+};
+
+const markupAuthor = (value) => {
+  const author = document.createElement("div");
+  const authorImg = document.createElement("img");
+  const authorInfo = document.createElement("div");
+  const authorName = document.createElement("p");
+  const comment = markupDescription(value.author.description);
+
+  author.className = "author";
+  authorImg.className = "author_img";
+  authorInfo.className = "author_info";
+  authorName.className = "author_name";
+
+  authorImg.setAttribute("src", value.author.photo);
+  authorImg.setAttribute("alt", "author photo");
+  authorName.textContent = value.author.name;
+
+  authorInfo.append(authorName);
+  authorInfo.append(comment);
+  author.append(authorImg);
+  author.append(authorInfo);
+
+  return author;
+};
+
+const markupHeader = () => {
```

Comment:
this function has a duplicate in all markup files too, please move it to separate file.